woocommerce/woocommerce
# Remove Product Collection wrapping div

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





In https://github.com/woocommerce/woocommerce/pull/51420 I raised a problem that when setting fixed width, the Product Collection becomes centered in Editor but left aligned in frontend (comment: https://github.com/woocommerce/woocommerce/pull/51420#pullrequestreview-2329977511). @octaedro fixed that by wrapping a block in a `div` in here: https://github.com/woocommerce/woocommerce/pull/51420#discussion_r1777282053.

I made a review and tests and merged the PR on behalf of @octaedro but didn't notice this `div` impacts other, more common flow by narrowing down Product Collection for example in a default setting in Product Catalog (even if the width is set to Fill!). So in this PR I'm removing this particular change to fix the main flow. Later we can think of improving the alignment when setting the fixed width.

Editor | Frontend - before | Frontend - after
-- | -- | --
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/7dfc5d1d-5272-4230-8a56-d0ba93d9b4a1"> | <img width="1727" alt="image" src="https://github.com/user-attachments/assets/6a8f1140-ee2b-41cc-8a65-6007fcfdcc7c"> | <img width="1728" alt="image" src="https://github.com/user-attachments/assets/b89ae17a-6ac6-40aa-a48a-bc5c770452ed">



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Editor
2. Reset Product Catalog template to default
3. Go to the frontend `/shop`
4. Expected: Product Collection is wide and nicely aligned with other elements of the page (see screenshot above) not shrinked



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
